backend/tests/conftest.py

```diff
@@ -164,6 +222,56 @@ async def title(title_dict):
 async def title_with_language(title, language_eng):
     await title.languages.add(language_eng)
     yield title
+    await title.languages.clear()
+    await title.tags.clear()
```

Comment:
While the title.languages.clear() is fine, the others are unnecessary and redundant with `title` fixture's teardown.